### PR TITLE
feat : Add Divider and its storybook #5

### PR DESCRIPTION
**Pull request by Hina-softwareEngineer**
Added the Divider Component

### CODE REVIEW

File: src/components/Divider/StyledDivider.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import styled, { css } from 'styled-components'
+// import { borderSizes } from '../common'
```
Comment (jpalacio0612) [inline src/components/Divider/StyledDivider.js]: can you remove this comment?

**Review by jpalacio0612 — CHANGES_REQUESTED** [review]
Great job!, just leave you a little comment. Thank you.

File: src/components/Divider/StyledDivider.js (hunk 1 above)
Comment (Hina-softwareEngineer) [reply src/components/Divider/StyledDivider.js]: Sure. This pull request is related to #5 issue.

**Review by jpalacio0612 — APPROVED** [review]
Great!

**Pull request merged**